pkgs/by-name/no/notion-app/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  unzip,
+  testers,
+  libplist,
+  jq,
+  writeText,
+  runCommand,
+}:
+let
+  info =
+    (builtins.fromJSON (builtins.readFile ./info.json))."${stdenvNoCC.targetPlatform.system}"
+      or (throw "notion-app: unsupported system ${stdenvNoCC.targetPlatform.system}");
```

Comment:
This shouldn't be necessary as it won't even try to evaluate on unsupported systems

Reply:
If I don't add this, ci will fail

Reply:
CI shouldn't even try to build this package if the platform it's building for isn't listed in `meta.platforms` — it is then your responsibility to set `meta.platforms` correctly

Reply:
https://github.com/NixOS/nixpkgs/actions/runs/12467905710/job/34798133430

Here is a example about this. I'm sure I set meta.platforms well.